lib/ansible/modules/packaging/os/yum.py

```diff
@@ -149,7 +149,8 @@
   enable_plugin:
     description:
       - I(Plugin) name to enable for the install/update operation.
-        The enabled plugin will not persist beyond the transaction.
+        The enabled plugin will not persist beyond the transaction. This description does not seem to enough. I tried this for yum-version
```

Comment:
@rajput1991 Thanks for this PR. Could you please file an issue which describes your problem with appropriate playbooks and console log ? The given description does not makes sense right now.

Reply:
> @rajput1991 Thanks for this PR. Could you please file an issue which describes your problem with appropriate playbooks and console log ? The given description does not makes sense right now.

Sure @AKasurde